compose/config/config.py

```diff
@@ -208,7 +208,7 @@ def version(self):
                 'Version in "{}" is invalid - it should be a string.'
                 .format(self.filename))
 
-        if version == '1':
+        if version == '1' or not all((n.isdigit() for n in version.split('.'))):
```

Comment:
nit: Any special reason for the extra parentheses inside `all()` ?

Reply:
Was passing the collection as a generator to `all()`. Uh! 🤦‍♂️ , Just realized it's actually not required. Let me update this ASAP. Thanks for catching this.